src/sentry/search/moresnuba/backend.py

```diff
@@ -59,24 +62,25 @@ def get_queryset_modifiers(
         date_from=None,
         date_to=None,
     ):
-        # We override this function to remove status and active_at
-        qs_builder_conditions = super(MoreSnubaSearchBackend, self).get_queryset_modifiers(
-            projects,
-            environments,
-            sort_by,
-            limit,
-            cursor,
-            count_hits,
-            paginator_options,
-            search_filters,
-            date_from,
-            date_to,
-        )
-
-        del qs_builder_conditions["status"]
-        del qs_builder_conditions["active_at"]
-
-        return qs_builder_conditions
+        with Hub.current.start_span(op="func", description="override_get_queryset_modifiers"):
```

Comment:
I don't see any metric about the actual query execution time. Is that intentional ?